crates/turbopack-tests/tests/snapshot/imports/subpath-imports/input/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "subpath-imports",
+  "imports": {
+    "#foo": "./foo.js",
+    "#dep": "dep",
+    "#conditional": {
+      "import": "./import.mjs",
+      "require": "./require.cjs"
+    },
+    "#pattern/*.js": "./*.js"
```

Comment:
Can imports field values refer to other imports fields? e.g.

```json
"#dep": "dep",
"#otherdep": "#dep",
```

Reply:
Nope. Running this in node gets me:

```
node:internal/errors:478
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '/Users/jridgewell/src/vercel/nextpack/turbo/crates/turbopack-tests/tests/snapshot/imports/subpath-imports/input/node_modules/' imported from /Users/jridgewell/src/vercel/nextpack/turbo/crates/turbopack-tests/tests/snapshot/imports/subpath-imports/input/package.json
    at new NodeError (node:internal/errors:387:5)
    at legacyMainResolve (node:internal/modules/esm/resolve:242:9)
    at packageResolve (node:internal/modules/esm/resolve:839:14)
    at resolvePackageTargetString (node:internal/modules/esm/resolve:420:16)
    at resolvePackageTarget (node:internal/modules/esm/resolve:471:12)
    at packageImportsResolve (node:internal/modules/esm/resolve:686:26)
    at moduleResolve (node:internal/modules/esm/resolve:895:21)
    at defaultResolve (node:internal/modules/esm/resolve:1115:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:841:30) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```